docs/src/app/components/pages/components/SelectField/ExampleCustomLabel.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import SelectField from 'material-ui/lib/SelectField';
+import MenuItem from 'material-ui/lib/menus/menu-item';
+
+export default class SelectFieldCustomLabelExample extends React.Component {
```

Comment:
how about an exact match: `ExampleCustomLabel` and also in the Page component again, the exact name: `ExampleCustomLabel`?